src/App.jsx

```diff
@@ -42,18 +41,38 @@ function App() {
   useEffect(() => {
     fetchData()
   }, [])
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
+
+  const removeTodo = (id) => {
+    setTodoList(todoList.filter(todo => todo.id !== id));
```

Comment:
It'd be good to go ahead and get the `removeTodo` and `addTodo` functions to use your Airtable database.

Just a recommendation! I believe it isn't required until the final project.